README.md

```diff
@@ -602,7 +602,7 @@ Which parser to use.
 Both the `babylon` and `flow` parsers support the same set of JavaScript
 features (including Flow). Prettier automatically infers the parser from the
 input file path, so you shouldn't have to change this setting.
-
+   
```

Comment:
Not a big deal, but if you manage to figure out where these trailing spaces come from it would be nice if we could remove them :)

Reply:
That's just my naive docblock regex, I realized that `require('jest-docblock').parseWithComments()` does a `.trim()` which removes wanted left-padding. I'll probably do a PR there to fix that.

Reply:
@lydell you might appreciate this regex fu! https://github.com/facebook/jest/pull/4576/files#diff-b5f6adcc40fabaaa89acc7ec660da205